src/monexamine.cpp

```diff
@@ -229,8 +232,10 @@ void monexamine::mount_pet( monster &z )
     z.add_effect( effect_ridden, 1_turns, num_bp, true );
     if( z.has_effect( effect_tied ) ) {
         z.remove_effect( effect_tied );
-        item rope_6( "rope_6", 0 );
-        g->u.i_add( rope_6 );
+        if( z.tied_item ) {
+            g->u.i_add( *z.tied_item, 0 );
```

Comment:
This `0` argument is being interpreted as `should_stack = false`.  Is that what you intended?